crates/router/src/core/payment_methods/utils.rs

```diff
@@ -0,0 +1,857 @@
+                            builder,
+                            supported_connectors,
+                        )
+                    {
+                        agg_or_nodes_for_mandate_filters.push((
+                            connector_eligible_for_mandates_node,
+                            cgraph::Relation::Positive,
+                            cgraph::Strength::Strong,
+                        ))
+                    }
+                }
+            }
+
+            // Non Prominent Mandate flows
+            let payment_type_non_mandate_value_node = builder.make_value_node(
+                cgraph::NodeValue::Value(dir::DirValue::PaymentType(
+                    euclid::enums::PaymentType::NonMandate,
+                )),
+                None,
+                None::<()>,
+            );
+            let payment_type_setup_mandate_value_node = builder.make_value_node(
+                cgraph::NodeValue::Value(dir::DirValue::PaymentType(
+                    euclid::enums::PaymentType::SetupMandate,
+                )),
+                None,
+                None::<()>,
+            );
+
+            let non_major_mandate_any_node = builder
+                .make_any_aggregator(
+                    &[
+                        (
+                            payment_type_non_mandate_value_node,
+                            cgraph::Relation::Positive,
+                            cgraph::Strength::Strong,
+                        ),
+                        (
+                            payment_type_setup_mandate_value_node,
+                            cgraph::Relation::Positive,
+                            cgraph::Strength::Strong,
+                        ),
+                    ],
+                    None,
+                    None::<()>,
+                    None,
+                )
+                .map_err(KgraphError::GraphConstructionError)?;
+
+            agg_or_nodes_for_mandate_filters.push((
+                non_major_mandate_any_node,
+                cgraph::Relation::Positive,
+                cgraph::Strength::Strong,
+            ));
+
+            let agg_or_node = builder
+                .make_any_aggregator(&agg_or_nodes_for_mandate_filters, None, None::<()>, None)
+                .map_err(KgraphError::GraphConstructionError)?;
+
+            agg_nodes.push((
+                agg_or_node,
+                cgraph::Relation::Positive,
+                cgraph::Strength::Strong,
+            ));
+
+            // Capture Method filter
+            config
+                .0
+                .get(connector.as_str())
+                .or_else(|| config.0.get("default"))
+                .and_then(|inner| match pmt.payment_method_type {
+                    api_enums::PaymentMethodType::Credit | api_enums::PaymentMethodType::Debit => {
+                        if let Ok(Some(capture_method_filter)) =
+                            construct_capture_method_node(builder, inner, &pmt.payment_method_type)
```

Comment:
Why are we adding capture method filters for cards only?

Reply:
Added it for all PMT